sorbet/rbi/shims/test_runner.rbi

```diff
@@ -0,0 +1,11 @@
+# typed: true
+
+module Test
+  module Unit
+    module RubyLsp
+      class TestRunner < ::Test::Unit::UI::TestRunner
+        include Kernel # for `method`
```

Comment:
(or do we prefer to put on the class?)

Reply:
Nope: https://github.com/test-unit/test-unit/blob/master/lib/test/unit/ui/testrunner.rb#L6

But I see it's [fine](https://sorbet.run/#%23%20typed%3A%20true%0Aextend%20T%3A%3ASig%0A%0Aclass%20MyClass%0A%20%20def%20foo%0A%20%20%20%20bar%28%26method%28%3Abaz%29%29%0A%20%20end%0A%20%20%0A%20%20def%20bar%0A%20%20end%0Aend) on sorbet.run so will try figure out what's going on.